docker-bake.hcl

```diff
@@ -0,0 +1,21 @@
+variable "JEKYLL_ENV" {
+  default = "development"
+}
+
+group "default" {
+  targets = ["release"]
+}
+
+target "release" {
+  target = "release"
+  args = {
+    JEKYLL_ENV = JEKYLL_ENV
+  }
+  no-cache-filter = ["upstream-resources"]
```

Comment:
Thanks to the `no-cache-filter` there is no need to use `docker build --no-cache`. Just this specific stage will be invalidated.